### ISSUE TITLE
Compatibility with packwerk

### ISSUE THREAD

**Issue by glampr** [astrauka/TestRSpec#52]
When using packwerk, a rails project can have multiple "spec" folders, so the approach to find the spec root folder does not work out of the box.

Specifically what is not working is any `require` directives the spec file might declare, as this assume that the working directory of the command is always Rails root.

Is it possible to:
- either specify multiple "spec" folders and choose which ever is found first at each run?
- or specify a new setting project/rails root, and if it exists always consider that as project root?

**Comment by oleg-vinted** [comment]
Before we can jump into solutions, I want to understand the problem better. We also use Packwerk and we always run specs from the Rails root directory, e.g. `bin/rspec packages/foo/spec/foo/bar_spec.rb`.

* Can you give more information about how your project structure is set up with Packwerk? This could be a simplified `tree` command output.
* How does your Rails application know about Packwerk packages? How does it discover / integrate them?

**Comment by glampr** [comment]
This is what the project folder looks like:

```
project-root
+ spec
+ packs
  + pack1
    + spec
  + pack2
    + spec
```

Whenever I try to run a spec from pack1, because of the spec folder configuration, the actual commands that gets executed is this:

```
cd /path/to/project-root/packs/pack1 && /path/to/.rvm/bin/rvm-auto-ruby -S rspec spec/models/a_model_in_pack1.rb:199
```

instead of 

```
cd /path/to/project-root && /path/to/.rvm/bin/rvm-auto-ruby -S rspec packs/pack1/spec/models/a_model_in_pack1.rb:199
```

Can this be fixed with altering some configuration?

**Comment by oleg-vinted** [comment]
I've actually been using a patched version of this plugin, I'll publish the changes in a branch later today. It addresses this exact issue.

I don't think this can be fixed with just configuration.

**Comment by OlegSmelov** [comment]
Sorry for the delay. Here's the branch with the patch: https://github.com/astrauka/TestRSpec/compare/feature/packwerk_support

Can you remove the package from Package Control and install it from GitHub - that branch specifically? I'd like to test the changes before publishing them.

**Comment by glampr** [comment]
Sorry for the delay!
Just tested this and it works just fine with packwerk!

Let's publish this! 😄

**Comment by OlegSmelov** [comment]
Fix released in [v2.3.0](https://github.com/astrauka/TestRSpec/releases/tag/v2.3.0).
